pkg/parser/parser_test.go

```diff
@@ -1478,6 +1478,12 @@ func fnm:{}any
     return {a:1{b:2}}
 end
 `: `line 3 column 16: expected map key, found "{"`,
+		`a := {"x": true}
+print a
+`: `line 1 column 7: expected map key, found "x"`,
+		`a := {"end": true}
+print a
+`: `line 1 column 7: expected map key, found "end"`,
```

Comment:
These tests all seem needlessly verbose for what they are testing. The following would seem to work just as well and are otherwise valid evy program (you could just use `a :=` instead of `print` since you are testing a parse error, so there is no need for `a` to actually be used with a followup `print`, but you also don't need a var anyway and can just have a valid program apart from the map literal):
```
        inputs := map[string]string{
                `print {a:1{b:2}}`:    `line 1 column 11: expected map key, found "{"`,
                `print {"x": true}`:   `line 1 column 8: expected map key, found "x"`,
                `print {"end": true}`: `line 1 column 8: expected map key, found "end"`,
        }
 ```

Reply:
yeah, way nicer. Thanks for looking so carefully. fixed.